README.md

```diff
@@ -2,6 +2,20 @@
 
 Icons, fonts, colors, components and all user interface items for OSRD project.
 
+# Build Instructions
+
+1. Go to the project's `root` directory in your terminal
```

Comment:
I personally find this sentence confusing: I'd try to find a directory named "root", but there is none.

In general I think it's fine to assume that the user is initially already in the project's directory.

Reply:
Yup. Agreed. Fixed.